### DISCUSSION TITLE
[Feature] fliter star rating

### DISCUSSION THREAD

**Discussion by benjaminv2** [immich-app/immich#11813] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

rating is very useful new feature,but i can't find out where to filter my rating.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by C-Otto** [comment]
@stumpigit is working on it: https://github.com/immich-app/immich/discussions/11775#discussioncomment-10339622

**Reply by stumpigit** [reply]
Thanks @C-Otto for the pointer. Exactly! The functionality is practically complete. Just implement the tests and then off we go... 

![starratingfilter](https://github.com/user-attachments/assets/bfd0bc08-e3b8-4a84-97cf-1d2a129437db)

**Comment by stumpigit** [comment]
View also #11777
